docs/data/charts/zoom-and-pan/zoom-and-pan.md

```diff
@@ -55,3 +55,18 @@ While the `zoom` prop is an array of objects that define the zoom state for each
 - **end**: The ending percentage of the zoom range.
 
 {{"demo": "ZoomControlled.js"}}
+
+## Zoom filtering
+
+You can filter the zooming behavior by setting the `zoom.filterMode` prop on the axis config.
+
+It allows the `secondary` axis to adapt to the data in the visible part of the `primary` axis.
+
+The `filterMode` will affect the axis opposite of the one with the filter mode set.
+
+- **keep** (default): The secondary axis will keep its original range.
+- **discard**: The secondary axis will discard values outside its view and adapt to the visible part of the primary axis.
```

Comment:
> You can connect the zoom of multiple axes by setting the `zoom.filterMode` prop on the axis config.

this is not true though, if an axis has its own zoom, then `filterMode` won't work on it, hence it doesn't connect the zoom of multiple axis.

Reply:
> it doesn't connect the zoom of multiple axis

Take this charts for example. The x-axis could impact the two y-axis

![image](https://github.com/user-attachments/assets/c52560f1-afce-42f4-9831-bb8770e0c211)

> if an axis has its own zoom, then filterMode won't work on it

Yes, but I can't think about a real usecase where I would like to have that

Reply:
What I meant to say is that, since only one axis has a zoom, it sounds odd if you say it connects multiple zooms. Because there is only one.  

We could go for something in the lines of

```You can make the zoom of an axis affect one or more axes extremums by setting the `zoom.filterMode` prop on the axis config.```

Reply:
Looks good :+1: